rasa/nlu/training_data/formats/rasa.py

```diff
@@ -35,7 +36,18 @@ def read_from_json(self, js: Dict[Text, Any], **_) -> "TrainingData":
 
         training_examples = []
         for ex in common_examples:
-            msg = Message.build(**ex)
+            # taking care of custom entries
+            other_values = {
+                key: value
+                for key, value in ex.items()
+                if key not in [TEXT, INTENT, ENTITIES]
+            }
```

Comment:
why do we need this?

Reply:
because we want to pass the inputs to the right places, i.e., test to test and intent to intent etc., but the data can include other things, e.g., `{"confidence": 0.8}` and we don't want to ignore that input.

Reply:
Oh, now I see what you mean 👍Let make a code suggestion